pkg/controller/kogitoapp/resource/deployment_config.go

```diff
@@ -32,6 +34,33 @@ const (
 	defaultReplicas = int32(1)
 	// ServiceAccountName is the name of service account used by Kogito Services Runtimes
 	ServiceAccountName = "kogito-service-viewer"
+
+	envVarInfinispanServerList     = "SERVER_LIST"
```

Comment:
shouldn't this env be a list?

Reply:
No, not at all. The name of the var has "list" on it because  the values can be a list. It's defined by the runtime, not us: https://quarkus.io/guides/infinispan-client#quarkus-infinispan-client_configuration